### PR TITLE
File to base64

### PR DESCRIPTION
**Pull request by erezh31**
Store file as base64 list in Demisto

### CODE REVIEW

File: Scripts/FileToBase64List/FileToBase64List.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            data = zlib.compress(data)
+
+    return base64.b64encode(data)
+
+
+def main():
+    LIST_NAME = demisto.args()['listName']
+    TO_ZIP = (demisto.args()['zipFile'] == 'yes')
+
+    res = demisto.executeCommand("getFilePath", {"id": demisto.args()['entryId']})
```
Comment (Itay4) [inline Scripts/FileToBase64List/FileToBase64List.py]: use `demisto.getFilePath()`

File: Scripts/FileToBase64List/FileToBase64List.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    file_path = res[0]['Contents']['path']
+
+    file_base64 = get_file_data(file_path, TO_ZIP)
+
+    res = demisto.executeCommand("createList", {"listName": LIST_NAME, "listData": file_base64})
+    if isError(res):
+        return res
+
+    return 'success store list ' + LIST_NAME + ' size: ' + str(len(file_base64))
```
Comment (Itay4) [inline Scripts/FileToBase64List/FileToBase64List.py]: please improve human readable

File: Scripts/FileToBase64List/FileToBase64List.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  name: listName
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'no'
+  description: Zip the file before store?
+  isArray: false
+  name: zipFile
+  predefined:
```
Comment (Itay4) [inline Scripts/FileToBase64List/FileToBase64List.yml]: use `true` and `false`

File: Scripts/FileToBase64List/FileToBase64List.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+script: '-'
+system: false
+tags:
+- Utility
+- list
+timeout: '0'
+type: python
+runas: DBotWeakRole
+runonce: true
+fromversion: 4.1.0
```
Comment (Itay4) [inline Scripts/FileToBase64List/FileToBase64List.yml]: why is that?

File: Scripts/FileToBase64List/FileToBase64List.py (hunk 2 above)
Comment (erezh31) [reply Scripts/FileToBase64List/FileToBase64List.py]: what do you mean? I have nothing to add here..

File: Scripts/FileToBase64List/FileToBase64List.yml (hunk 4 above)
Comment (erezh31) [reply Scripts/FileToBase64List/FileToBase64List.yml]: createList added in version 4.1

**Comment by erezh31** [comment]
@Itay4 CR fixed

**Comment by kirbles19** [comment]
Edited file returned to @erezh31 in Slack.

**Pull request merged**